src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/UserForm.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@(
+Html
+    .DevExtreme()
+    .Form<LoginModelRecord>()
+    .OnInitialized("userFormInitialized")
+    .ID("UserForm")
+    .Items(items =>
+    {
+        items.AddCustomFormTextBox(model => model.Username);
+
+        items

```

Comment:
Crete Reusable component for Previous and Next

Reply:
Done